packages/xforms-engine/src/parse/model/RootNamespaceDeclaration.ts

```diff
@@ -0,0 +1,28 @@
+import { XMLNS_PREFIX } from '@getodk/common/constants/xmlns.ts';
+import { escapeXMLText } from '../../lib/xml-serialization.ts';
+
+export class RootNamespaceDeclaration {
+	private readonly serializedXML: string;
+
+	constructor(
+		readonly prefix: string | null,
+		readonly namespaceURI: string | null
+	) {
+		if (prefix == null) {
+			// We intentionally omit the default namespace declaration, which is
+			// consistent with both Collect and Enketo. Including it may technically
+			// be more "correct", but consistency ensures we don't introduce subtle
+			// problems in any namespace-aware usage downstream.
```

Comment:
Hmmm. So, I just added this comment because I think it could be confusing without. But I'm also a little bit cautious about this being a blanket omission without any logic _or_ explanation about the expected default. I am assuming here that the default namespace will always be XForms (`http://www.w3.org/2002/xforms`). But I could see making any of these changes:

- Explicitly check for that assumption (don't omit if some other default namespace)
- Explicitly assert that assumption (fail form load if some other default namespace)
- Expand this comment to mention that assumption, and let weird forms do weird things
- Test in Collect and Enketo to be sure

Reply:
Agree the comment covers everything important then! Thank you.